spec/intl.html

```diff
@@ -1338,6 +1338,101 @@ <h1>CalendarEraYear ( _calendar_, _dateLike_ )</h1>
+        </emu-clause>
+
+        <emu-clause id="sec-temporal-calendardateday">
+          <h1>CalendarDateDay ( _calendar_, _date_ )</h1>
+          <p>The abstract operation CalendarDateDay performs implementation-defined processing to find the day for the date corresponding to _date_ in the context of the calendar represented by the string _calendar_ and returns it.</p>
+        </emu-clause>
+
+        <emu-clause id="sec-temporal-calendardatedayofweek">
+          <h1>CalendarDateDayOfWeek ( _calendar_, _date_ )</h1>
+          <p>The abstract operation CalendarDateDayOfWeek takes a date-like object _date_ and returns the day of the week in the calendar represented by _calendar_.</p>
+        </emu-clause>
+
+        <emu-clause id="sec-temporal-calendardatedayofyear">
+          <h1>CalendarDateDayOfYear ( _calendar_, _date_ )</h1>
+          <p>The abstract operation CalendarDateDayOfYear takes a date-like object _date_ and returns the day of the year in the calendar represented by _calendar_.</p>
+        </emu-clause>
+
+        <emu-clause id="sec-temporal-calendardateweekofyear">
+          <h1>CalendarDateWeekOfYear ( _calendar_, _date_ )</h1>
+          <p>The abstract operation CalendarDateWeekOfYear takes a date-like object _date_ and returns the week of the year in the calendar represented by _calendar_.</p>
+        </emu-clause>
+
+        <emu-clause id="sec-temporal-calendardatedaysinweek">
+          <h1>CalendarDateDaysInWeek ( _calendar_, _date_ )</h1>
+          <p>The abstract operation CalendarDateDaysInWeek takes a date-like object _date_ and returns the number of days in the given week in the calendar represented by _calendar_.</p>
+        </emu-clause>
+
+        <emu-clause id="sec-temporal-calendardatedaysinmonth">
+          <h1>CalendarDateDaysInMonth ( _calendar_, _date_ )</h1>
+          <p>The abstract operation CalendarDateDaysInMonth takes a date-like object _date_ and returns the number of days in the given month in the calendar represented by _calendar_.</p>
+        </emu-clause>
+
+        <emu-clause id="sec-temporal-calendardatedaysinyear">
+          <h1>CalendarDateDaysInYear ( _calendar_, _date_ )</h1>
+          <p>The abstract operation CalendarDateDaysInYear takes a date-like object _date_ and returns the number of months in the given year in the calendar represented by _calendar_.</p>
+        </emu-clause>
+
+        <emu-clause id="sec-temporal-calendardatemonthsinyear">
+          <h1>CalendarDateMonthsInYear ( _calendar_, _date_ )</h1>
+          <p>The abstract operation CalendarDateMonthsInYear takes a date-like object _date_ and returns the number of months in the given year in the calendar represented by _calendar_.</p>
+        </emu-clause>
```

Comment:
May want to note that this will normally be 12 for most calendars, but some lunisolar calendars have "leap months" which may return 13.

Reply:
Key points to note:
* Not all calendars will always return 12 months. For example, Coptic returns 13
* Not all years will return the same value, e.g. lunisolar calendars will return one more month in leap years.